### PR TITLE
wait for child procs complete

### PR DESCRIPTION
**Pull request by rink1969**

### CODE REVIEW

File: src/process.rs [hunk 1]
```diff
@@ -109,11 +109,12 @@ impl Processes {
             Some(pid) => {
                 let pid_str = pid.to_string();
                 let args = vec!["-9", &pid_str];
-                Command::new("kill")
-                    .args(args)
-                    .spawn()
-                    .expect("kill command failed to start");
-                info!("{} stopped", name);
+                let status = Command::new("kill").args(args).status();
```
Comment (TheWaWaR) [inline src/process.rs:112]: Why not `spawn()` now?

File: src/process.rs [hunk 2]
```diff
@@ -109,11 +109,12 @@ impl Processes {
...
+                if status.is_ok() {
+                    info!("kill {} {} exit with {}", name, &pid_str, status.unwrap());
+                } else {
+                    info!("kill {} {} failed", name, &pid_str);
+                }
```
Comment (TheWaWaR) [inline src/process.rs]: Should check if the exit status is success? 
https://doc.rust-lang.org/std/process/struct.ExitStatus.html
``` rust
match status {
    Ok(exit_status) if exit_status.success() => {
        info!("kill {} {} exit with {}", name, &pid_str, exit_status.code().unwrap_or(0));
    },
    _ => info!("kill {} {} failed", name, &pid_str);
}
```

File: src/process.rs (hunk 1 above)
Comment (rink1969) [reply src/process.rs:112]: 等待子进程执行完，否则某些情况下会有问题

Comment (TheWaWaR) [reply src/process.rs:112]: ok

**Review by TheWaWaR — APPROVED** [review]
OK to me now.

**Review by wzf2020 — APPROVED** [review]
ok

**Pull request merged**